docs/src/components/Guidelines/helpers.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { imageWrapperClass } from "gatsby-remark-images/constants";
+import { css } from "styled-components";
+import { mediaQueries as mq } from "@kiwicom/orbit-components";
+
+import { Content } from ".";
+
+const isImage = obj => obj.props?.children?.props?.className === imageWrapperClass;
+
+export const extractContent = data => {
+  return React.Children.toArray(data).reduce(
+    (acc: Content, cur) => {
+      if (isImage(cur)) acc.images.push(cur);
+      else acc.content = cur;
+
+      return acc;
+    },
+    { images: [], content: null },
+  );
+};
+
+const setBorders = isMobile => ({ coloredBorder, type, theme }) => {
+  const position = isMobile ? "top" : "left";
+
+  if (coloredBorder) {
+    if (type === "do")
+      return css`
+        border-${position}: 3px solid ${theme.orbit.paletteGreenNormal};
+      `;
+
+    if (type === "dont") {
+      return css`
+        border-${position}: 3px solid ${theme.orbit.paletteRedNormal};
+      `;
+    }
+  }
+
+  return css`
+    border-${position}: 3px solid ${theme.orbit.paletteInkLight};
+  `;
+};
+
+export const resolveBorders = () => {
+  return css`
+    ${setBorders(true)};
+    ${mq.desktop(css`
+      border-top: none;
+      ${setBorders(false)};
+    `)}
+  `;
+};
```

Comment:
`setBorders` introduces complexity, you don't need it.
```suggestion
export const resolveBorders = ({ theme, type, coloredBorder }) => {
  let borderColor: string;
  if (coloredBorder) {
    if (type === "do") {
      borderColor = theme.orbit.paletteGreenNormal;
    }
    if (type === "dont") {
      borderColor = theme.orbit.paletteRedNormal;
    }
  } else {
    borderColor = theme.orbit.paletteInkLight;
  }
  return css`
    border-top: 3px solid ${borderColor};
    ${mq.desktop(css`
      border-top: none;
      border-left: 3px solid ${borderColor};
    `)}
  `;
};
```

Reply:
not agree. I prefer a declarative way, rather than imperative with let. In my opinion, your suggestion looks more complex (edited)